container/lxd/manager.go

```diff
@@ -55,8 +54,6 @@ type containerManager struct {
 // containerManager implements container.Manager.
 var _ container.Manager = (*containerManager)(nil)
 
-var generateCertificate = func() ([]byte, []byte, error) { return lxdshared.GenerateMemCert(true) }
```

Comment:
I believe we used to have this because time spent generating a certificate during the test suite was a significant chunk of time. I respect that we probably don't trigger this right now, but do we know if it impacted 'time go test'?

Reply:
This was never actually patched out, but my prior patch for certs added ```GenerateClientCertificate``` which wraps the upstream method. This can be patched at need.

I checked and it does take 10+ seconds. So I have removed an unnecessary call to it in one of the unit tests.